aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -22,39 +18,69 @@ We exercise empathy in our voice through situational tones that speak to each co
-- **Respect the user and their time**
-  - Establish context
-  - Lead with what is most important
-  - Explain using terminology consistent with the UI
-  - Point the user directly to a resolution. Do not use "_contact your
-    administrator_" or "_contact HPE_" &mdash; the administrator is likely
-    the user and the support contact could be a channel partner. Instead,
-    provide a direct means to connect the user with the party able to resolve
-    the issue.
-- **Inspire trust and confidence**
-  - Use informative, concise language
-  - State facts without coloring them in opinions
-  - Avoid charged words such as "_error_"
-  - Focus on the solution, avoid casting blame on the user or software
-  - Focus on what is needed to be accomplished, avoid personal pronouns
-    (e.g. I, you, he, she, it, we, they, me, him, her, us, them)
-- **Ensure the voice is human and humane**
-  - Read your message out loud to another person. Is this how you would say it out loud? Does the other person feel this is how two people would converse?
+## Writing Effective Content
+
+A design speaks for itself. 
+
+It communicates the interaction of the user experience; how a task is accomplished.
+
+Content supplements design.
+
+It communicates the goal of the user experience; why a task should be accomplished.
+
+We can create meaningful user experiences by writing clarity into our content; what we say and how we say it. Clarity simplifies complex ideas and connects one idea to the next to bridge the gap between user and application, task and goal. 
+
+We can use a few simple tactics to create clarity: 
+
+### Accessibility 
+
+Write with clear language to communicate clear ideas.
+
+Write to users in plain language, grounded in positive ability and action. Plain language requires less cognitive effort from users, improving time to action. Write at a United States grade 7 reading level to create straightforward, succinct, and consistent content.
+
+Use the [Hemingway App](https://hemingwayapp.com/) to check the readability of content.
+
+Identify and communicate key points by separating and refining ideas to their essence. Words can often be reordered and refined to create simple and neat sentences.
+
+Read your message aloud to hear if it sounds like natural conversation. Content that sounds natural is easier to read and understand.
+
+### Objectivity
+
+Anchor content in fact and objectivity, free of bias and judgement.
+
+Content based in fact fosters clarity and reliability. Content based in opinion introduces ambiguity.
+
+Consider the connotations of content. Words like ***Error*** can unintentionally imply that the user is at fault and convey judgement of their actions.
+
+Consider content from the user's perspective and goals, not what the UI can do.
```

Comment:
An example of a "good" and "bad" here might be helpful. If "Error" is bad, how could the message read instead?

Reply:
I agree that this section needs examples. My plan is to break that work out into a separate ticket next week and create examples in Figma that contextualize how the text works with and complements the UI.

Thoughts? Should I mark this PR as a draft until I create and include examples?

Reply:
I am fine moving forward with the PR as is and have the examples come later on! Can you just make sure there is a ticket in the backlog for that work?

Reply:
Will do!

Reply:
Here's the [ticket](https://github.com/grommet/hpe-design-system/issues/2262) for example creation.